_content/gateways/thethingsindoor/faq.md

```diff
@@ -11,31 +11,52 @@ This page consolidates some of the most frequently asked questions on [The Thing
 If registering using `ttnctl`, please follow the convention of `eui-<your 64bit EUI in lower case for the hex alphabets>`.
 
-**Can I use any browser to configure the gateway?** 
+#### Q. Can I use any browser to configure the gateway?
 The WiFi config page has a small JS issue when working with Firefox. Chrome will work just fine. There will be an automatic firmware update soon that will fix this.
 
-**Q. I don't live in Europe or in the US, can I still use the gateway?**
+#### Q. I don't live in Europe or in the US, can I still use the gateway?
 
 Currently, the gateway only supports the EU868 and US915 frequency plans. Near the end of Q1, all frequency plans will be supported.
 
-**Q. How do I know if my Gateway is EU868 or US915?**
+#### Q. How do I know if my Gateway is EU868 or US915?
 
 Please have a look at the serial number on the back of your gateway.
 TMBH100**868**xxxxxx -> EU
 TMBH100**915**xxxxxx -> US
 
-**Q. Is registration in TTN Console required?**
+#### Q. Is registration in TTN Console required?
 
 Yes the registration is mandatory. For now, these gateways are identified as US915/EU868 based on their EUIs and a default frequency plan is sent out. But, with the official V3 support, you can select other supported frequency plans while you register your gateways as you would do with other gateways currently.
 
 
-**Q. How can I get the EUI of my gateway?**
+#### Q. How can I get the EUI of my gateway?
 
 The EUI of the gateway is NOT the WiFi MAC address printed on the back of the gateway but is derived from the first number on the top of the sticker below the QR code.This number can also be obtained from the WiFi Setup Page
     ![TTIG_EUI](TTIG_EUI.jpg)
 To derive this take the code for example *5BA0CB80042B* and insert *FFFE* to after the first 6 characters to make it a 64bit EUI such as *5BA0CBFFFE80042B*. This is the value to be entered in the Gateway ID field on the [console](https://console.thethingsnetwork.org).
+
+#### Q. I want to get in-depth insight/read debug messages of my gateway. Is that possible?
+
+Yes it's possible, but it requires opening the casing of your gateway. 
+
+> Please proceed with these steps only if you really know what you're doing and are aware that this may damage your gateway.
+
+The casing needs to be first opened by unscrewing two screws found below the top and the bottom panel. Then, connect a 3.3V UART-USB Interface such as an FTDI to your computer. 
+Connect the Rx (FTDI) to the Tx of the Gateway and ground pins.
+Use a serial terminal (Ex: `PUTTY` on Windows and `screen`/`minicom` on macOS/Linux) with a baudrate of 115200@8N to read the serial out. The connections are shown in the image below
+    ![TTIG_Serial](TTIG_Serial.jpg)
+
+
+#### Q. I want to delete and re-create (re-register) TTIG. Can I do that?
+
+This is definitely not recommended since you cannot recreate a gateway with the same EUI on the same network. We do this to prevent violation of uniqueness on the EUI.
```

Comment:
This step is **not recommended** as you cannot recreate a gateway with the same EUI on the same network. We do this to prevent violation of uniqueness on the EUI. So please be sure before deleting.